pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
```

Comment:
Here's what @deads2k and I are proposing now:

`rollback <name>`

We'll try and use `<name>` as an RC first, then a DC, and use whatever we find. The possibility of a DC and RC having the same name within a namespace seems extremely low, since RC names are `<config-name>-<version>`.

Both `rc/` and `dc/` prefixes would be supported.